openpype/hosts/houdini/plugins/publish/validate_abc_primitive_to_detail.py

```diff
@@ -73,6 +73,13 @@ def get_invalid(cls, instance):
         cls.log.debug("Checking Primitive to Detail pattern: %s" % pattern)
         cls.log.debug("Checking with path attribute: %s" % path_attr)
 
+        if not isinstance(output_node, hou.SopNode):
+            # In the case someone has explicitly set an Object
+            # node instead of a SOP node in Geometry context
+            # then for now we ignore.
+            cls.log.warning("No geometry output node found, skipping check..")
+            return
```

Comment:
This doesn't seem entirely accurate right? Should it ever allow a non sop node?

Reply:
Would you recommend using this conditional because it is more general? 

```
if not hasattr(output_node, "geometry"):
  # In the case someone has explicitly set an Object
  # node instead of a SOP node in Geometry context
  # then for now we ignore - this allows us to also
  # export object transforms.
  cls.log.warning("No geometry output node found, skipping check..")
  return
```

> P.S. this file doesn't relate to the goal of this enhancement, I only added because it kept throwing error 
because of this line `geo = output_node.geometryAtFrame(frame)`